src/processTargets/processSelectionType.ts

```diff
@@ -196,7 +202,7 @@ function processNonWhitespaceSequence(
       )
       .find((range) => range.contains(position));
     if (result == null) {
-      throw new Error("Couldn't find containing non whitespace sequence");
+      throw new Error(`Cannot find sequence defined by regex: ${regex}`);
     }
     return result;
   };
```

Comment:
I'm not totally sure if there's functionality specific to the non-whitespace scope that I'm ignoring when making this generic. Definitely consider that in the review

Reply:
Not really. I think this should work fine.